.github/workflows/ruby.yml

```diff
@@ -25,3 +25,4 @@ jobs:
       plugin: katello
       postgresql_container: ghcr.io/theforeman/postgresql-evr
       test_existing_database: false
+      foreman_version: refs/pull/10128/head
```

Comment:
Added this to run tests with the Foreman PR. Tests are green, except that our "React Tests" check seems to ignore the Foreman version specified.  However, `npm run test` passes locally (with the Foreman PR checked out):

```
Test Suites: 137 passed, 137 total
Tests:       584 passed, 584 total
Snapshots:   165 passed, 165 total
```